### DISCUSSION TITLE
Integrate the Win11 Volume Control

### DISCUSSION THREAD

**Discussion by ColaGrain** [valinet/ExplorerPatcher#4066] [Ideas]
On Win11, you can adjust the volume by hovering your cursor over the speaker icon and using the mouse wheel. I hope you can include this on the Win10 style taskbar.

**Comment by pyrates999** [comment]
I am running windows 10, and I have the volume icon set to use the windows 7 volume flyout, and can use the mouse wheel to adjust it.  It also works with the windows 10 volume flyout.

What flyout do you have it set to in windows 11?

**Reply by ColaGrain** [reply]
My sound setting is currently on Win10 flyout (default).

You can certainly adjust it using the mouse wheel. but in Win 10 but you have to click the icon first. Am I correct? 

The Win 11 version has the added functionality of being able to adjust the volume just by hovering over the icon.

**Reply by pyrates999** [reply]
If the code exists in windows 10, then maybe.
